javascript/javascript2/week3/warmup8.js

```diff
@@ -0,0 +1,8 @@
+let lastClickTime = 0;
+document.addEventListener("click", function() {
```

Comment:
There is a special event for double click :) 
 [You can read about it here](https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event)